srcpkgs/python-pex/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'python-pex'
+pkgname=python-pex
+version=1.4.6
+revision=1
+wrksrc="pex-${version}"
+noarch=yes
+build_style=python-module
+pycompile_module="pex"
+hostmakedepends="python-setuptools python3-setuptools python-Sphinx python3-Sphinx"
+depends="python python-wheel"
+short_desc="A library and tool for generating .pex (Python EXecutable) files (Python2)"
+maintainer="Daniel Santana <daniel@santana.tech>"
+license="Apache-2.0"
+homepage="https://github.com/pantsbuild/pex"
+distfiles="${PYPI_SITE}/p/pex/pex-${version}.tar.gz"
+checksum=b82c687b7f9099c6d76bc2bc5707c60e38bd741efa434846eac8c70e4f89ed29
+alternatives="
+ pex:pex:/usr/bin/pex2
+ pex:pex.1:/usr/share/man/man1/pex2.1"
+
+post_extract() {
+	rm -r pex.egg-info
+}
+
+post_build() {
+	cd docs
+	make man
+}
+
+post_install() {
+	vman docs/_build/man/pex.1 pex2.1
+	vlicense LICENSE
```

Comment:
Apache-2.0 doesn't need LICENSE

Reply:
Wow, I didn't know this one. Should I remove it from `minikube`, `telepresence` and `dockeru-machine-driver-kvm2` (the packages I'm maintaining) too?